<launch>
  
  <rosparam command="load" file="$(find cube_pick_place)/launch/params/planner_params.yml" ns="planner_parameters"/>

  <!-- <node name="moveit_pick_place_tests_node" pkg="cube_pick_place" type="moveit_pick_place_tests" output="screen" launch-prefix="terminator -x gdb -r-args"/> -->
  <node name="moveit_pick_place_tests_node" pkg="cube_pick_place" type="moveit_pick_place_tests" output="screen"/>

</launch>
